### PR TITLE
Added images and created slide animation

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,7 +1,2 @@
 <router-outlet class="router-content"></router-outlet>
```
Comment (Rementh) [inline src/app/app.component.html]: Do not style router-kotlet.

File: src/app/landing-page/slider/slider.component.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.img-container {
```
Comment (Rementh) [inline src/app/landing-page/slider/slider.component.scss]: No ale no po co to

Comment (Rementh) [reply src/app/landing-page/slider/slider.component.scss]: Dont understand

File: src/app/landing-page/slider/slider.component.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+@Component({
+  selector: 'slider',
+  templateUrl: './slider.component.html',
+  styleUrls: ['./slider.component.scss'],
+  animations: slideAnimation
+})
+
+export class SliderComponent {
+
```
Comment (Rementh) [inline src/app/landing-page/slider/slider.component.ts]: Unnecesary line break

File: src/styles.scss [hunk 4]
```diff
@@ -4,6 +4,10 @@
   color: #3b3b3b;
 }
 
+html, body {
+  overflow-x: hidden;
```
Comment (Rementh) [inline src/styles.scss]: Try not to set that kind of agressive styles on root elements like html or body. Just use it directly where needed. If nedded.

**Comment by ewelina-slepko** [comment]
Done!

**Pull request merged**